### PR TITLE
Adding Register action

### PR DESCRIPTION
**Pull request by websemantics**

### CODE REVIEW

File: src/User/Register/RegisterFormBuilder.php [hunk 1]
```diff
@@ -54,6 +54,18 @@ class RegisterFormBuilder extends FormBuilder
...
+    
+    /**
+     * The form actions.
+     *
+     * @var array
+     */
+    protected $actions = [
+        'register' => [
+            'button' => 'blue',
+            'text'   => 'Register'
```
Comment (RyanThompson) [inline src/User/Register/RegisterFormBuilder.php]: We have to use translatable strings in core man

Comment (websemantics) [inline src/User/Register/RegisterFormBuilder.php]: Totally, I wasn't sure how. Same has to be done to the Login form, ..

File: src/User/Register/RegisterFormBuilder.php [hunk 2]
```diff
@@ -54,6 +54,18 @@ class RegisterFormBuilder extends FormBuilder
...
         'last_activity_at'*/
     ];
+    
+    /**
+     * The form actions.
+     *
+     * @var array
+     */
+    protected $actions = [
+        'register' => [
```
Comment (RyanThompson) [inline src/User/Register/RegisterFormBuilder.php]: And really a pretty sure this can be blue and remove button => blue just to clean it up a smidge

Comment (websemantics) [inline src/User/Register/RegisterFormBuilder.php]: Cool, .. the magic from within

**Comment by RyanThompson** [comment]
Thank you!

**Pull request merged**